Documentation/Contributing/development-flow.md

```diff
@@ -97,147 +95,135 @@ rook
 │   ├── charts                    # Helm charts
 │   │   └── rook-ceph
 │   │   └── rook-ceph-cluster
-│   └── examples                  # Sample yaml files for Rook cluster
+│   └── examples                  # Sample manifestes to configure the cluster
 │
-├── cmd                           # Binaries with main entrypoint
+├── cmd
 │   ├── rook                      # Main command entry points for operators and daemons
 │
-├── design                        # Design documents for the various components of the Rook project
-├── Documentation                 # Rook project Documentation
-├── images                        # Dockerfiles to build images for all supported storage providers
+├── design                        # Design documents
+├── Documentation                 # Documentation that is published to rook.io
+├── images                        # Rook docker image sources
 │
 ├── pkg
 │   ├── apis
-│   │   ├── ceph.rook.io          # ceph specific specs for cluster, file, object
+│   │   ├── ceph.rook.io          # ceph specs used in the CRDs
 │   │   │   ├── v1
 │   ├── client                    # auto-generated strongly typed client code to access Rook APIs
 │   ├── clusterd
-│   ├── daemon                    # daemons for each storage provider
+│   ├── daemon                    # daemons for configuring ceph
 │   │   ├── ceph
 │   │   └── discover
-│   ├── operator                  # all orchestration logic and custom controllers for each storage provider
+│   ├── operator                  # all reconcile logic and custom controllers
 │   │   ├── ceph
 │   │   ├── discover
 │   │   ├── k8sutil
-│   │   └── test
-│   ├── test
 │   ├── util
 │   └── version
-└── tests                         # integration tests
-    ├── framework                 # the Rook testing framework
```

Comment:
this all info is good to have,

Reply:
Some extra comments got removed, I'll add some back. But I think not many details are needed since if they actually care about integration tests, they would dive into those source files.